### ISSUE TITLE
How are JSON files for single routes named?

### ISSUE THREAD

**Issue by kytrinyx** [octokit/routes#170]
The JSON file seems to be named after the name of the route, which I thought was based on the title in the developer docs, but I found a file that doesn't follow this pattern.

Would you help me understand which bit I'm missing?

- endpoint scope: Gists
- endpoint name: List user's gists
- route: `GET /users/:username/gists`
- documentation URL: https://developer.github.com/v3/gists/#list-a-users-gists

### Current specification

https://github.com/octokit/routes/blob/d8c3b5db9c9d1f7cef7bf2afbbf7e1382b309a8f/routes/gists/list-public-gists-for-the-specified-user.json#L5

### I expected

The developer documentation lists this as "List user's gists" not "List public gists for the specified user".

**Comment by kytrinyx** [comment]
I think I just figured out what's going on.

The `GET /gists` and `GET /users/:username/gists` both have the same documentation URL:
https://developer.github.com/v3/gists/#list-a-users-gists

I found the other endpoint in the delightfully named https://github.com/octokit/routes/blob/d8c3b5db9c9d1f7cef7bf2afbbf7e1382b309a8f/routes/gists/list-the-authenticated-users-gists-or-if-called-anonymously-this-will-return-all-public-gists.json

Which gave me a hint :) It looks like the name is the prose just preceding the route in the documentation.

**Comment by gr2m** [comment]
> It looks like the name is the prose just preceding the route in the documentation.

Exactly, it’s a somewhat hacky workaround. I’ve reached out to the docs team and asked them if we could make sure to have only a single route per section in the documentation, it would make the code for octokit/routes a lot easier

Right now each section is handled so that it can result in multiple route specification, the logic is here: https://github.com/octokit/routes/blob/master/lib/endpoint/find-routes.js the comments in there reference a few different cases for that
